packages/teams-js/test/public/clipboard.spec.ts

```diff
@@ -0,0 +1,261 @@
+import { errorLibraryNotInitialized } from '../../src/internal/constants';
+import { GlobalVars } from '../../src/internal/globalVars';
+import { DOMMessageEvent } from '../../src/internal/interfaces';
+import { clipboard } from '../../src/public';
+import { app } from '../../src/public/app';
+import { errorNotSupportedOnPlatform, FrameContexts } from '../../src/public/constants';
+import { Utils } from '../utils';
+
+/* eslint-disable */
+/* As part of enabling eslint on test files, we need to disable eslint checking on the specific files with
+   large numbers of errors. Then, over time, we can fix the errors and reenable eslint on a per file basis. */
+
+describe('clipboard', () => {
+  const dataToCopy: Blob = new Blob(['Mock data'], { type: 'text/plain' });
```

Comment:
it would be good to test each of the valid types, instead of choosing just one of them

Reply:
Added test cases for all the valid types in latest commit.